src/components/ui/table/table.tsx

```diff
@@ -8,6 +8,7 @@ interface TableProps<T extends { id: number }> {
   isRowClickable?: boolean
   overflowXAuto?: boolean
   overflowYHidden?: boolean
+  tableFrom?: string
```

Comment:
can we use something like `applyFixedColWidth` with boolean type, by default should be false

Reply:
Done :3

Reply:
can pls remove `tableFrom`

Reply:
oppss mb xD